### ISSUE TITLE
Tabbing away from terminal Emacs crashes tmux server

### ISSUE THREAD

**Issue by thomasjm** [tmux/tmux#2568]
### Issue description

To reproduce, just start a new `tmux` session and then type `emacs -nw`. Then, alt-tab away from the terminal window, and the entire `tmux` server will crash, also wiping out all other sessions. If you close emacs with `C-x C-c` without tabbing away, no crash happens--somehow this seems to be due to the terminal losing focus.

This has been happening to me for a while on a version of `tmux 3.2-rc3` I built. I just tried it on the most recent `3.2-rc3` branch and it still happens.

I think the relevant log lines are these:

```
1613368023.268372 writing key 0x1000000001 (FocusOut) to %0
1613368023.268375 fatal: invalid key modifiers: 1000000001
```

### Required information

Please provide the following information:

* tmux version (`tmux -V`): `tmux 3.2-rc3`
* Platform (`uname -sp`): `Linux x86_64`
* $TERM inside and outside of tmux (`echo $TERM`). Outside is `xterm-256color`. Inside is `screen-256color`.
* Logs from tmux (`tmux kill-server; tmux -vv new`). Attached.

[tmux-server-691307.log](https://github.com/tmux/tmux/files/5979696/tmux-server-691307.log)
[tmux-client-691305.log](https://github.com/tmux/tmux/files/5979697/tmux-client-691305.log)
[tmux-out-691307.log](https://github.com/tmux/tmux/files/5979698/tmux-out-691307.log)

**Comment by nicm** [comment]
Does this still happen if you build from master?

**Comment by thomasjm** [comment]
No, I can't repro on master.

**Comment by nicm** [comment]
OK thanks, I think it is fixed, I will update the RC from master soon

**Comment by nicm** [comment]
I have merged the fix into 3.2-rc branch now (0d28ee927421c46315bc0b47b5f8f49a8392efa4 -> 5c275c2a1a963876d4ac392067e42120417dbf43). Thanks!
